virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -59,7 +59,7 @@ endif::[]
 
 * To deploy {VirtProductName} in a disconnected environment, you must xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[configure Operator Lifecycle Manager on restricted networks].
 
-* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator.
+* When using a disconnected cluster on a restricted network, you must xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {VirtProductName} Operator. A proxy can also be used if the cluster is partially connected to a fully disconnected installation. See xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more information.
```

Comment:
I'm not sure what "cluster is partially connected to a fully disconnected installation" means

Reply:
I think it should be "A proxy can also be used if the cluster is partially connected."

Reply:
Thanks @fabiand - I got that phrase from the Jira record, just to let you know I did not make it up. :-) Changed it to: _A proxy can also be used if the cluster is partially connected._

Also, on the PR you created, yes, a rebase is going to be needed. We made a lot of major attribute changes in the doc set that required all our branches to be rebased recently. That should be completed today.